pwa/app/routes/event.$eventKey.tsx

```diff
@@ -168,8 +182,9 @@ export default function EventPage() {
     awards,
     teams,
     teamMedia,
+    colors,
     coprs,
-  } = useLoaderData<typeof loader>();
+  } = useLoaderData<Awaited<ReturnType<typeof loadData>>>();
```

Comment:
I don't know why, but after I added `colors` here, `coprs` became `unknown`. I tried everything I could think of to keep the generic type the same (`typeof loader`) but this is the only thing I could get working.